examples/e-commerce/package.json

```diff
@@ -17,7 +17,7 @@
     "@angular/platform-browser": "^6.0.2",
     "@angular/platform-browser-dynamic": "^6.0.2",
     "@angular/router": "^6.0.2",
-    "angular-instantsearch": "1.0.1",
+    "angular-instantsearch": "^2.0.4",
```

Comment:
I've kept the `^`. Any recommendation against it? @algolia/instantsearch-for-websites

Reply:
Nope not for the production dependencies.

Reply:
ok thanks for explaining why offline :)